Brokerages/Bitfinex/BitfinexBrokerage.Messaging.cs

```diff
@@ -577,7 +570,21 @@ private void EmitFillOrder(string[] entries)
                         new OrderFeeParameters(security, order));
                 }
 
-                OrderStatus status = fillQuantity == order.Quantity ? OrderStatus.Filled : OrderStatus.PartiallyFilled;
+                OrderStatus status = OrderStatus.Filled;
+                if (fillQuantity != order.Quantity)
+                {
+                    if (filling.ContainsKey(order.Id))
+                    {
+                        filling[order.Id] += fillQuantity;
+                    }
+                    else
+                    {
+                        filling.AddOrUpdate(order.Id, fillQuantity);
+                    }
+                    status = filling[order.Id] == order.Quantity
+                        ? OrderStatus.Filled
+                        : OrderStatus.PartiallyFilled;
+                }
```

Comment:
We can reduce the number of dictionary accesses here, something like this:
``` C#
                if (fillQuantity != order.Quantity)
                {
                    decimal totalFillQuantity;
                    filling.TryGetValue(order.Id, out totalFillQuantity);
                    totalFillQuantity += fillQuantity;
                    filling[order.Id] = totalFillQuantity;

                    status = totalFillQuantity == order.Quantity
                        ? OrderStatus.Filled
                        : OrderStatus.PartiallyFilled;
                }
```

Reply:
@StefanoRaggi It looks like it's now updated can we get it merged?